### PR TITLE
heading stye changed #622

### PR DESCRIPTION
**Pull request by DeeyaSingh**
Changed table format

### CODE REVIEW

File: floss/render/default.py [hunk 1]
```diff
@@ -208,14 +208,14 @@ def render_heading(heading, n, ostream, disable_headers):
...
         | FLOSS STATIC STRINGS (1337) |
-        -------------------------------
+        +-----------------------------+
     """
     if disable_headers:
         return
-    heading = f"| {heading} ({n}) |"
-    ostream.write(tabulate.tabulate([[heading]]))
+    heading = f"  {heading} ({n})  "
+    ostream.write(tabulate.tabulate([[heading]],tablefmt="psql"))
```
Comment (mr-tz) [inline floss/render/default.py:218]:
```suggestion
    ostream.write(tabulate.tabulate([[heading]], tablefmt="psql"))
```

File: floss/render/default.py [hunk 2]
```diff
@@ -208,14 +208,14 @@ def render_heading(heading, n, ostream, disable_headers):
...
+        +-----------------------------+
         | FLOSS STATIC STRINGS (1337) |
-        -------------------------------
+        +-----------------------------+
     """
     if disable_headers:
         return
-    heading = f"| {heading} ({n}) |"
-    ostream.write(tabulate.tabulate([[heading]]))
+    heading = f"  {heading} ({n})  "
```
Comment (mr-tz) [inline floss/render/default.py:217]: How does one space vs. two spaces look like. Which one do you find nicer?

Comment (DeeyaSingh) [reply floss/render/default.py:217]: I think one space might look better

Comment (mr-tz) [reply floss/render/default.py:217]: I think so. Can you update and paste an example output of a run here, please?

Comment (DeeyaSingh) [reply floss/render/default.py:217]: Hey, I have tried varying the spaces in the code, but I seem to be getting the same output every time. Can you please guide me with this

Comment (DeeyaSingh) [reply floss/render/default.py]: So, I removed the spaces completely does that look better?

**Comment by DeeyaSingh** [comment]
image
 
This is the output I'm getting

File: floss/render/default.py [hunk 3]
```diff
@@ -208,14 +208,14 @@ def render_heading(heading, n, ostream, disable_headers):
...
         | FLOSS STATIC STRINGS (1337) |
-        -------------------------------
+        +-----------------------------+
     """
     if disable_headers:
         return
-    heading = f"| {heading} ({n}) |"
-    ostream.write(tabulate.tabulate([[heading]]))
+    heading = f"{heading} ({n})"
+    ostream.write(tabulate.tabulate([[heading]], tablefmt="psql"))
```
Comment (williballenthin) [inline floss/render/default.py:218]: nice solution!

**Review by williballenthin — APPROVED** [review]

**Comment by williballenthin** [comment]
thank you @DeeyaSingh this is great!

**Pull request merged**